dts/bindings/radio/radio.yaml

```diff
@@ -0,0 +1,26 @@
+---
+title: Radio Base Structure
+id: radio
+version: 0.1
+
+description: >
+    This binding gives the base structures for all radio devices
+properties:
+    reg:
+      type: array
+      description: mmio register space
+      generation: define
+      category: required
+
+    interrupts:
+      type: array
+      category: optional
+      description: required interrupts
```

Comment:
required in description, optional in category

Reply:
that's bug